### ISSUE TITLE
During new card creation keyboard shortcuts are still detected

### ISSUE THREAD

**Issue by balintbarna** [nextcloud/deck#6252]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**
When I create a new card I have to type in the name of the card. During this keyboard shortcuts are still detected if the mouse is hovered over another card, leading to all kinds of unintended behaviour such as accidentally archiving cards or marking them done/not done.

**To Reproduce**
1. Go to a board with a list that includes some cards already
2. Click on the plus button for that list and start typing "hell"
3. Hover your mouse pointer over one of the existing cards
4. Continue typing an "o" to finish your card called "hello"

**Expected behavior**
The placeholder new card will contain the text "hello"

**Current behavior**
The card being hovered over toggles its completed state

**Client details:**
 - Brave Browser (chromium)

**Nextcloud version:** (see Nextcloud admin page)
29.0.2

**Where did you install Nextcloud from:**
NixOS

**Comment by annettekusma** [comment]
This also happens to me, browser Firefox, Nextcloud 29.0.1

It is **hard to detect the behaviour** and really nasty. I accidentally deleted several cards before I realized that I even did it. I realized the screen looked differently, but since the app is supposed to help me remember, I don't remember myself all the cards there, so I couldn't tell what or if anything had been deleted in that blink of an eye. There is no "Undo" pop up or other indication, like graphic refreshing of the list or so. I repeatedly thought I had pressed my mouse too lightly so that the cursor was not in the field for naming the new card. Only when I had a list with only one card, did I finally manage to grasp the problem. 

As described, click the plus button to create a new card. Intuitively one might want to click inside the textfield. That's not necessary, but even if you don't click there, your mouse curser at the plus button would be very close to the field, so it is very natural to just slide it downwards or sidewards before typing. Then you most likely have accidentally hovered over a card, which might be hard to detect if you are not aware and/or as soon as you start typing and completion suggestions appear on top of that card. And more, even if you move the mousebutton past other cards, the last card does not get unactive, so even if you intentionally move your mousebutton to a blank area of the screen, a card may still feel hovered over. 

So this behaviour is not merely irritating, but it very nastily **let's cards dissapear without ones knowledge or possibility to even understand which card was affected.**

**Comment by mejo-** [comment]
I agree that the shortcuts can be very irritating and have heard from several people that they got confused by cards disappearing etc.. I would suggest to completely replace the single-key shortcuts like <kbd>A</kbd> with shortcuts that include a meta key (e.g. <kbd>Ctrl</kbd>-<kbd>Alt</kbd>-<kbd>A</kbd>). @marcoambrosini for design team input and @grnd-alt.

**Comment by marcoambrosini** [comment]
> (e.g. Ctrl-Alt-A)

The challenge with this is that we would override other native shortcuts that users might still want to use.

I'm currently testing keyboard navigation to try these shortcuts and it does not seem to work for me. Is it broken or maybe I'm not testing this properly?
